content/topics/intro-to-typescript/_index.md

```diff
@@ -5,11 +5,28 @@ ready: true
 title: Introduction to Typescript
 ---
 
+### What is Typescript
+
+- TypeScript is a superset of JavaScript that adds types as well as other language features.
+- Typescript was created to make it easier to write code that is compatible with the JavaScript language.
+- Strong static typing, compilation, and object-oriented programming are all features of TypeScript that has made it a popular language.
+
+#### The TypeScript Compiler
+
+- TypeScript must be compiled into JavaScript for execution, the code gets compiled to JavaScript by the TypeScript Compiler.
```

Comment:
I have read mostly about the transpiler in TS, perhaps rather us transpile by a transpiler?